Makefile.am

```diff
@@ -41,10 +41,10 @@
 # OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 #
 
-WARN_CFLAGS = -Wall -Wextra -Werror
+WARN_CFLAGS = -Wall -Wextra -Wformat=2 -Werror
 
 ACLOCAL_AMFLAGS = -I cmulocal
-AM_CFLAGS = $(WARN_CFLAGS) @PERL_CCCDLFLAGS@ $(GCOV_CFLAGS)
+AM_CFLAGS = -O3 $(WARN_CFLAGS) @PERL_CCCDLFLAGS@ $(GCOV_CFLAGS)
```

Comment:
This `-O3` should really be up to the the person compiling the software.  That person might want `-O2`, so it should not be implied by the build system.  The default for Autoconf/Automake is `-O2`, so one can rely on this default. Besides, `-O3` is already present on many other places: `libcrc32_la_CFLAGS = -O3 $(AM_CFLAGS) $(CFLAG_VISIBILITY)`, and many `__attribute__((optimize("-O3")));` in twom.c, util.c, dlist.c and others.

Reply:
This commit is not to be merged and is only here to help this PR detect more errors.